package.json

```diff
@@ -40,7 +40,7 @@
         "sprintf-js": "^1.1.2",
         "static-kdtree": "^1.0.2",
         "sweetalert": "^2.1.2",
-        "three": "^0.90.0",
+        "three": "^0.111.0",
         "three-orbit-controls": "^82.1.0",
         "three-transform-controls": "^1.0.4",
```

Comment:
We should also make sure that the orbit controls and transform controls are compatible with the new THREE.js version - see the notion at the end of https://threejs.org/docs/index.html#manual/en/introduction/Import-via-modules